doc/tutorials/workshop/content/exercises/03-creating-your-first-scan.md

```diff
@@ -480,7 +480,7 @@ Once you’re done with the extraction.
 
 > **NOTE**
 > 
-> Using the [`oc-compliance`](https://github.com/JAORMX/oc-compliance) plugin 
+> Using the [`oc-compliance`](https://github.com/openshift/oc-compliance) plugin 
```

Comment:
There is also another mention of `oc-compliance` on line 302, would you mind fixing it there too?